### PR TITLE
fix: support laravel 7

### PR DESCRIPTION
**Pull request by atymic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,10 +6,9 @@ php:
     - 7.4
 
 env:
-    - LARAVEL_VERSION=5.6.*
```
Comment (antonkomarev) [inline .travis.yml:9]: Why did you removed 5.6 and 5.7 versions from testing?

Comment (atymic) [reply .travis.yml:9]: They are well out of support, little point in testing them now. Meant to remove the from the `composer.json` as well.

**Comment by antonkomarev** [comment]
Thanks for the contribution. I've updated changelog file:
https://github.com/laravel-notification-channels/rocket-chat/blob/master/CHANGELOG.md

**Comment by antonkomarev** [comment]
I want to think couple more days about public API and then we will be ready to publish v0.2.0.

**Comment by antonkomarev** [comment]
Resolved #12

**Pull request merged**